### DISCUSSION TITLE
Updating a Joomla 3.10.11 website to Astroid/Template 2.6.0+ with Child template ability

### DISCUSSION THREAD

**Discussion by Chacapamac** [templaza/astroid-framework#241] [Q&A] [answered]
I’m testing updating a quite complicated website with still Joomla 3.10.11 to Astroid/Template 2.6.0+ with Child template ability.

**Step 1:**
**I update Astroid framework 2.5.20 to 2.6.2**  _(with Child template ability)_

**Note:**
Before testing the following I make sure to delete manually any residual compiled css in the normal “templates” folder to be sure they don't have any influence as the copy of this folder now in the “media” folder is where compiled css file are cache. I also empty all Astroid/Joomla/Browser caches.

**Results and first questions.** _______________________________
**Q1.** 
Website is fine, but curiously some local background images set in my “custom.scss” are not showing because they are now call from the “media” folder?
![back-images-media-folder](https://user-images.githubusercontent.com/1907276/200633500-4de1c6d9-49c9-447c-a507-26ad12ef0ee2.jpg)
— Original  working background images calls from my “custom.scss” example:
`background: url(../../../images/x-construction/logos-site/vertical-background-takuma-typo-81-572.png);`
As you can see, these images should be load by the normal **joomlaroot > images** folder, not “media”???

**Q2.** 
The folder “scss” containing also my optional “custom.scss” are not move/copy to folder “media” but now compiled in the “**media > css**” folder. This way All my custom scss still work.
I know the “scss” folder is moved to “media” when I update my **Astroid Zero Template 2.5.19** _(without Child Template ability)_ in my next Update Step to to **Zero 2.6.0+** (with Child template ability) . 
I guess this behavior is to keep everything working even if you update only the Framework?

**Comment by sonvnn** [comment, marked as answer]
Hi @Chacapamac 

> Website is fine, but curiously some local background images set in my “custom.scss” are not showing because they are now call from the “media” folder?

Yes, you have to update background-image url because css was change from `templates` to `media`. You should prepend `../../` to url to make sure it works. 
Solution change `background: url(../../../images/x-construction/logos-site/vertical-background-takuma-typo-81-572.png);` to `background: url(../../../../../images/x-construction/logos-site/vertical-background-takuma-typo-81-572.png);`

> The folder “scss” containing also my optional “custom.scss” are not move/copy to folder “media” but now compiled in the “media > css” folder. This way All my custom scss still work.
I know the “scss” folder is moved to “media” when I update my Astroid Zero Template 2.5.19 (without Child Template ability) in my next Update Step to to Zero 2.6.0+ (with Child template ability) .
I guess this behavior is to keep everything working even if you update only the Framework?

I think you have to edit your custom.scss manually. I don't like using code to update the template provider scss file. Because a lot of exceptions can arise when meddling in other people's code

As you know when Joomla update from Joomla 4.1 to 4.2. There are many extensions are not compatible with it and they have to update to compatible with latest version. Astroid is same in this situation, template provider have to upgrade their code to compatible. 

Thanks & Best Regards,
Sonny

**Comment by Chacapamac** [comment]
Thank You for the info, I undersyand. Now I will test to update the Template Zero to see if I have pronlems.
